go/grpc/registry/entity.proto

```diff
@@ -36,13 +36,15 @@ message DeregisterResponse {
 
 message EntityRequest {
     bytes id = 1;
+    int64 height = 2;
```

Comment:
General question applicable to everywhere that has a height param.  Should it consistently be `height = 1` so that when requests grow more fields, it still is consistent?  Or does it not matter?

Reply:
Aren't we getting rid of protobufs anyway?

Reply:
At some point, it's been on our todo list forever...